pkg/kubelet/rkt/rkt.go

```diff
@@ -52,11 +52,12 @@ import (
 )
 
 const (
-	RktType = "rkt"
+	RktType                      = "rkt"
+	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
 	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "0.13.0"
-	recommendedRktBinVersion = "0.13.0"
+	minimumRktBinVersion     = "1.2.1"
```

Comment:
IMHO ```minimumRktBinVersion     = "1.2.1"```, so in this case we should drop ```stage-1-image``` support, because ```rkt``` > 1.0  is missing this flag, or if we keep support we should put at least 0.16 `rkt` bin